reflex/components/__init__.py

```diff
@@ -157,6 +157,7 @@
 box = Box.create
 center = Center.create
 circle = Circle.create
+cond = Cond.create
```

Comment:
The `cond` function is a bit unique, as it can either return `Cond.create` if the arguments are components, or something else when using conditional props. For example in the case of `rx.text(color=rx.cond(State.foo, "blue", "green"))`.

You can see the function defined here: https://github.com/reflex-dev/reflex/blob/76b8af3b42a618939c1fd7dd8af5dfaad9f5906b/reflex/components/layout/cond.py#L81

So we are importing this function already at the top when we do `from .layout import *`, so there's no need to redefine it again here.

Reply:
Hi @picklelo. It is because mypy complains about it like this...

![image](https://github.com/reflex-dev/reflex/assets/583807/41f0168e-5f28-4dd2-b13f-a44a7fa43246)

![image](https://github.com/reflex-dev/reflex/assets/583807/ae97e8a9-7244-48c7-ad1f-f3c58300c07f)

Reply:
I like to not have mypy complain about anything. :wink:

Reply:
I will have a look at what mypy is making of cond.py/def conf() and get back to you.